atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1642,9 +1642,18 @@ private void putMetadataForTablesInternal(
     }
 
     private void putMetadataWithoutChangingSettings(final TableReference tableRef, final byte[] meta) {
```

Comment:
Not directly related to this PR but seems odd that the above call to putMetadataForTablesInternal 1) is called by a method called internalPutMetadataForTables 2) does the actual schema mutation of the cf as well as what I expect it to do which is mutate the metadata table.